core/templates/dev/head/pages/header_css_libs.html

```diff
@@ -1,8 +1,11 @@
 <link rel="stylesheet" type="text/css" media="screen"
       href="https://fonts.googleapis.com/css?family=Capriola|Rubik|Roboto|Material+Icons|Open+Sans:400,600">
-
+{% if MINIFICATION or not DEV_MODE %}
+<link rel="stylesheet" type="text/css" media="screen"
```

Comment:
thought of making another PR with changes you asked and (may be) asked by other reviewers :)